Chap_API_Server.tex

```diff
@@ -1300,7 +1300,21 @@ \subsection{\code{pmix_server_fencenb_fn_t}}
 \adviceimplend
 
 \advicermstart
-The host will receive a single call for each collective operation. It is the responsibility of the host to identify the nodes containing participating processes, execute the collective across all participating nodes, and notify the local \ac{PMIx} server library upon completion of the global collective.
+The host will receive a single call for each collective operation. It is the responsibility of the host to identify the nodes containing participating processes, execute the collective across all participating nodes, and notify the local \ac{PMIx} server library upon completion of the global collective. Data received from each node \emph{shall} be simply concatenated to form an aggregated unit, as shown in the following example:
+
+\cspecificstart
+\begin{codepar}
+uint8_t *blob1, *blob2, *total;
+size_t sz_blob1, sz_blob2, sz_total;
+
+sz_total = sz_blob1 + sz_blob2;
+total = (uint8_t*)malloc(sz_total);
+memcpy(total, blob1, sz_blob1);
+memcpy(\&total[sz_blob1], blob2, sz_blob2);
+\end{codepar}
+\cspecificend
+
+Note that the ordering of the data \emph{blobs} does not matter.
```

Comment:
Why do we emphasis *blobs* here? I do not see anything similar in the rest of the standard.